src/skmultiflow/drift_detection/hddm_w.py

```diff
@@ -113,7 +113,7 @@ def add_element(self, prediction):
             self.total.independent_bounded_condition_sum = 1
         else:
             self.total.EWMA_estimator = self.lambda_option * \
-                prediction + aux_decay_rate * self.total.EWMA_estimator
+                                        prediction + aux_decay_rate * self.total.EWMA_estimator
```

Comment:
Is there a reason to adopt this indentation style? Perhaps some "PEP*"? The previous version looked good to me, but I am not sure whether or not this new style is the preferred one according to community guidelines.

Reply:
I used [autopep8](https://github.com/hhatto/autopep8) to format the file. Do you think it's an issue?

Reply:
I don't think so. As long as the linting tests pass, I'm cool with that! :D